index.bs

```diff
@@ -11302,6 +11310,175 @@ The [=remote end steps=] given |session| and |command parameters| are:
 </div>
 
 
+## The webExtension Module ## {#module-webExtension}
+
+The <dfn export for=modules>webExtension</dfn> module contains functionality for
+managing and interacting with web extensions.
+
+### Definition ### {#module-webExtension-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+WebExtensionsCommand = (
+  webExtension.Install,
+  webExtension.Uninstall
+)
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+WebExtensionsResult = (
+  webExtension.InstallResult
+)
+</pre>
+
+### Types ### {#module-webExtension-types}
+
+#### The webExtension.Extension Type #### {#type-webExtension-Extension}
+
+<pre class="cddl remote-cddl local-cddl">
+webExtension.Extension = text
+</pre>
+
+The <code>webExtension.Extension</code> type represents a web extension id within a browser.
+
+### Commands ### {#module-webExtension-commands}
+
+#### The webExtension.install Command #### {#command-webExtension-install}
+
+The <dfn export for=commands>webExtension.install</dfn> command installs a web extension for the session.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      webExtension.Install = (
+         method: "webExtension.install",
+         params: webExtension.InstallParameters
+      )
+
+      webExtension.InstallParameters = {
+         extensionData: webExtension.ExtensionData,
+      }
+
+      webExtension.ExtensionData = (
+         webExtension.ExtensionArchivePath /
+         webExtension.ExtensionBase64Encoded /
+         webExtension.ExtensionPath
+      )
```

Comment:
Hm, I didn't really notice it before but shouldn't it just be the following?

```suggestion
      webExtension.InstallParameters = {
         webExtension.ExtensionData,
      }

      webExtension.ExtensionData = (
         webExtension.ExtensionArchivePath /
         webExtension.ExtensionBase64Encoded /
         webExtension.ExtensionPath
      )
```

I don't think that we need this extra level of abstraction, and could still add other fields like `temporary` later on to `webExtension.InstallParameters`. @jgraham can you please have a look?

Reply:
Alright then lets leave it as is.